src/events/SDL_mouse.c

```diff
@@ -661,17 +661,47 @@ SDL_SendMouseWheel(SDL_Window * window, SDL_MouseID mouseID, float x, float y, S
     return posted;
 }
 
+static SDL_MouseWheelSource latest_source = SDL_MOUSEWHEEL_SOURCE_LAST;
+
+static void
+SDL_SetPanSource(SDL_MouseWheelSource source) {
+    latest_source = source;
+}
+
+static SDL_MouseWheelSource
+SDL_GetPanLastSource() {
+    return latest_source;
+}
+
+int
+SDL_SendPanDelta(
+        SDL_Window * window,
+        SDL_MouseID mouseID,
+        double delta,
+        SDL_PanAxis axis
+) {
+    SDL_Mouse *mouse = SDL_GetMouse();
```

Comment:
Can you provide link in the SDL api ? Can this return NULL ? If so it is not handled.

Reply:
This is not documented or defined that I can find, but the implementation returns a reference to a static var and this makes the same assumptions of code that was in this file before (assumed mouse is non-null, blind/unchecked deref)